addons/account_edi_ubl_cii/models/account_edi_common.py

```diff
@@ -13,27 +13,17 @@
 # -------------------------------------------------------------------------
 UOM_TO_UNECE_CODE = {
     'uom.product_uom_unit': 'C62',
-    'uom.product_uom_dozen': 'DZN',
+    'uom.product_uom_pack_6': 'HD',
     'uom.product_uom_kgm': 'KGM',
     'uom.product_uom_gram': 'GRM',
     'uom.product_uom_day': 'DAY',
     'uom.product_uom_hour': 'HUR',
     'uom.product_uom_ton': 'TNE',
     'uom.product_uom_meter': 'MTR',
-    'uom.product_uom_km': 'KMT',
-    'uom.product_uom_cm': 'CMT',
     'uom.product_uom_litre': 'LTR',
-    'uom.product_uom_cubic_meter': 'MTQ',
     'uom.product_uom_lb': 'LBR',
-    'uom.product_uom_oz': 'ONZ',
     'uom.product_uom_inch': 'INH',
     'uom.product_uom_foot': 'FOT',
-    'uom.product_uom_mile': 'SMI',
-    'uom.product_uom_floz': 'OZA',
-    'uom.product_uom_qt': 'QT',
-    'uom.product_uom_gal': 'GLL',
-    'uom.product_uom_cubic_inch': 'INQ',
-    'uom.product_uom_cubic_foot': 'FTQ',
```

Comment:
As commented before, dangerous for existing databases that are using these units of measure, as the matching is done solely in the code and cannot be updated by the user